src/index.tsx

```diff
@@ -3,9 +3,12 @@ import * as ReactDOM from 'react-dom';
 import registerServiceWorker from './registerServiceWorker';
 import App from './App';
 import './index.css';
+import { IntlProvider } from 'react-intl';
 
 ReactDOM.render(
-  <App />,
+  <IntlProvider locale="en">
```

Comment:
Just curious, do we need to put `locale="en"` for now because otherwise it would break for non-en-default browsers?

Reply:
I think this is telling the it what translations to use. so in the future we should get this from the user's location/browser to determine, right now just hard code english since thats all we have anyway